test/AddressListTest.php

```diff
@@ -95,6 +95,18 @@ public function testCanAddManyAddressesAtOnce()
         $this->assertTrue($this->list->has('fw-announce@lists.zend.com'));
     }
 
+    public function testLosesParensInName()
+    {
+        $header = '"Supports (E-mail)" <support@example.org>';
+
+        $to = Header\To::fromString('To:' . $header);
+        $addressList = $to->getAddressList();
+        $address = $addressList->get('support@example.org');
+        $this->assertEquals('Supports', $address->getName());
+        $this->assertEquals('E-mail', $address->getComment());
+        $this->assertEquals('support@example.org', $address->getEmail());
```

Comment:
not testing the `toString`?

Reply:
Probably should! I was primarily ensuring that the various accessors returned what was expected.